### ISSUE TITLE
NSOIL8 not read properly when running sys_cpl != 1

### ISSUE THREAD

**Issue by aubreyd** [NCAR/wrf_hydro_nwm_public#571]
## Current Behavior (issue identified by @TimLahmers... thanks Tim!)
It looks like the ZSOIL8 data field (soil layer discretization) is currently being read into WRF-Hydro through a hrldas routine. When running NOT under hrldas (so sys_cpl != 1, e.g., LIS coupling), these routines are not called so the ZSOIL8 variable is not being set.

The ZSOIL8 variable appears to be read in hrldas_cpl_HYDRO_ini (maybe based on the specs in namelist.hrldas?) and then ALSO in the orchestrator in the hydro.namelist read, but the place where those values are mapped to the nlst object in the orchestrator is commented out (see lines 639-642 in https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/OrchestratorLayer/config.f90).

## Possible Solutions (courtesy of @danrosen25)
1 - Activate those commented out lines in the orchestrator when sys_cpl != 1
2 - Call the hrldas init routines from the NUOPC cap

**Comment by danrosen25** [comment]
@rcabell 
I plan to make the following change if you agree?
```
if(sys_cpl .ne. 1) then
    nlst(did)%NSOIL = NSOIL
    allocate(nlst(did)%ZSOIL8(NSOIL))
    nlst(did)%ZSOIL8 = ZSOIL8
endif
```
[https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/OrchestratorLayer/config.f90#L639](https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/OrchestratorLayer/config.f90#L639)

**Comment by rcabell** [comment]
Yep, that looks fine to me. @kafitzgerald or @aubreyd, any concerns?

> @rcabell
> I plan to make the following change if you agree?
> 
> ```
> if(sys_cpl .ne. 1) then
>     nlst(did)%NSOIL = NSOIL
>     allocate(nlst(did)%ZSOIL8(NSOIL))
>     nlst(did)%ZSOIL8 = ZSOIL8
> endif
> ```
> 
> https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/OrchestratorLayer/config.f90#L639
